packages/portal-proto/src/pages/404.tsx

```diff
@@ -0,0 +1,31 @@
+import { NextPage } from "next";
+import Head from "next/head";
+import { UserFlowVariedPages } from "@/features/layout/UserFlowVariedPages";
+import { headerElements } from "@/features/user-flow/workflow/navigation-utils";
+import Link from "next/link";
+
+const Custom404Page: NextPage = () => {
+  return (
+    <UserFlowVariedPages {...{ indexPath: "/", headerElements }}>
+      <Head>
+        <title>Page Not Found</title>
+        <meta
+          property="og:title"
+          content="GDC Data Portal Page Not Found"
+          key="gdc-page-not-found"
+        />
+      </Head>
+      <div className="flex flex-col justify-center items-center h-full p-8 gap-2">
+        <h1 className="text-2xl">Page Not Found</h1>
+        <h2>Sorry, we couldn&apos;t find the page you were looking for.</h2>
```

Comment:
Got `Page should contain a level-one heading` warning from the axe devtools otherwise.

Reply:
Looking at [https://dequeuniversity.com/rules/axe/4.1/page-has-heading-one](https://dequeuniversity.com/rules/axe/4.1/page-has-heading-one) wonder if adding role="heading" aria-level="1" would be appropriate.

Reply:
This is kind of similar to setting the first one as `h1`. However, the problem is that if don't make the next line as `h2`, then I get `Heading levels should only increase by one`. This is because the footer contains `h3` tag.
<img width="937" alt="Screenshot 2024-01-18 at 2 41 19 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/101295912/832dd291-fdc5-4be3-8274-9e1a5d941ed7">

Reply:
Starting with an `<h1>` for headings makes sense, but it's not required by WCAG or HTML 5.2. While having a logical heading hierarchy is recommended, not having an `<h1>` doesn't violate WCAG.

However, it's not ideal to skip heading levels (like going from `<h2>` to `<h4>`). This may make the structure less clear, but it's not a WCAG failure.

A WCAG failure occurs when text that looks like a heading isn't marked as one, or vice versa. WCAG emphasizes using the most appropriate structure for conveying information, including headings. While it doesn't force you to use headings, if you visually style text to indicate a section, it should be marked up as a heading.

Additionally, WCAG insists on a logical order for headings. While skipping levels is allowed, heading levels should align with the content's hierarchy. In simpler terms, less important headings should have lower hierarchy levels.

In our case, what we're not aligning with is a logical heading order. Also, we incorrectly labeled some text as a heading when it wasn't one.

To address these issues, refrain from marking "National Cancer Institute at the National Institutes of Health" as a heading. Additionally, ensure that the titles of the footer navigation are at the `<h2>` level.

![Screenshot 2024-01-19 at 14 21 57](https://github.com/NCI-GDC/gdc-frontend-framework/assets/100392615/61729940-513b-4713-afc0-18821ae62cdc)

Reply:
I would suggest fixing these in this PR. It does increase the scope, but we might as well fix it now.

Reply:
I have done as discussed. Please verify it once.